webapp/src/components/common/Matrix/components/MatrixGrid/index.tsx

```diff
@@ -141,6 +145,105 @@ function MatrixGrid({
     return true;
   };
 
+  const handleKeyDown = useCallback(
+    (event: GridKeyEventArgs) => {
+      // Fill selection with value (Ctrl+Shift+Enter)
+      if (event.shiftKey && event.ctrlKey && event.key === "Enter") {
+        if (
+          gridSelection.current?.range ||
+          gridSelection.rows.length > 0 ||
+          gridSelection.columns.length > 0
+        ) {
+          const userInput = prompt(t("matrix.fillSelection.numberPrompt"));
+
+          if (userInput === null) {
+            return;
+          }
+
+          const value = Number(userInput);
```

Comment:
Checks if `userInput` is empty because `Number("")` gives `0`.

Reply:
thanks